roles/core/node/tasks/install_license_pkg.yml

```diff
@@ -77,4 +79,6 @@
   with_items:
     - "{{ scale_install_gpfs_adv.files.0.path | basename }}"
     - "{{ scale_install_gpfs_crypto.files.0.path | basename }}"
-  when: '"gpfs.license.std" not in scale_gpfs_license_rpm'
+  when:
+    - '"gpfs.license.std" not in scale_gpfs_license_rpm'
+    - '"gpfs.license.da" not in scale_gpfs_license_rpm'
```

Comment:
I _think_ this code logic could be simplified...   

If i understand this correctly, we are searching for rpms above, and it's either there or not.   If we find it, wouldn't we want to install it?  Is there a case when we would not?   Otherwise this when clause will keep checking different strings in various license ... would this list grow?  

Suggest something like this: 

```
- name: install | Add GPFS Dependent License RPMs to list
  vars:
    current_rpm: "{{ gpfs_path_url }}/{{ item.files.0.path | basename }}"
  set_fact:
    scale_install_license_rpms: "{{ scale_install_license_rpms + [ current_rpm ] }}"
  with_items:
    - "{{ scale_install_gpfs_adv }}"
    - "{{ scale_install_gpfs_crypto }}"
  when:
    - item.matched | bool
```

and the changes
```diff
Victors-MBP-2:~ vhu$ diff new.yml  orig.yml
3c3,4
<     current_rpm: "{{ gpfs_path_url }}/{{ item.files.0.path | basename }}"
---
>     license_rpm: "{{ scale_install_gpfs_license.files.0.path | basename }}"
>     current_rpm: "{{ gpfs_path_url }}/{{ item }}"
7,8c8,9
<     - "{{ scale_install_gpfs_adv }}"
<     - "{{ scale_install_gpfs_crypto }}"
---
>     - "{{ scale_install_gpfs_adv.files.0.path | basename }}"
>     - "{{ scale_install_gpfs_crypto.files.0.path | basename }}"
10c11,12
<     - item.matched | bool
---
>     - '"gpfs.license.std" not in scale_gpfs_license_rpm'
>     - '"gpfs.license.da" not in scale_gpfs_license_rpm'
```

Reply:
But it may create issue if some one has kept 2 version of same package in the directory .

Reply:
Not sure I understand that comment.  Do you mean if the user is trying to install StandardEdition but has copied the adv and crypto rpms into that path? So we don't want to install them? 

Or, do you mean the index 0, and taking the first rpm?  If index 0, we are still taking the first one, index 0, it's just moved to the set of `current_rpm: "{{ item.files.0.path | basename }}"` 

What this suggestion is doing is just getting rid of itemizing the license names in the `when` clause:
```
   - '"gpfs.license.std" not in scale_gpfs_license_rpm'
   - '"gpfs.license.da" not in scale_gpfs_license_rpm'
```

Reply:
I'm OK either way, just a suggestion to not have to itemize those license checks in multiple places in the code.